src/js/base/helpers.cy.js

```diff
@@ -0,0 +1,127 @@
+import { View } from 'marionette';
+
+import hbs from 'handlebars-inline-precompile';
+
+import { testTs } from 'helpers/test-timestamp';
+import formatDate from 'helpers/format-date';
+
+context('Handlebars helpers', function() {
+  specify('Match text formatting', function() {
+    const MatchTextView = View.extend({
+      template: hbs`
+        <div class="test-null">{{matchText "Patient Name" null}}</div>
+        <div class="test-match">{{matchText "Patient Name" "Patient"}}</div>
+        <div class="test-noescape">{{matchText "<span style='color: green'>Patient</span> Name" "Patient" noEscape=true}}</div>
```

Comment:
Is this how `noEscape=true` would potentially be used? I don't fully understand the use case for it.

Reply:
Yeah so I think perhaps a more relevant example might be something like `Patient Name <span class="role">Role</span>` we've done something like this in the past, and without noEscape the "<span>" html would end up as part of the text.